src/ResizeSensor/ResizeSensor.ts

```diff
@@ -1,27 +1,40 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import _debounce from 'lodash/debounce';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import _debounce = require('lodash/debounce');
 import { props, t, ReactChildren } from '../utils';
-import _ResizeSensor from 'css-element-queries/src/ResizeSensor';
+import _ResizeSensor = require('css-element-queries/src/ResizeSensor');
 
 export const Props = {
   children: ReactChildren,
   onResize: t.Function,
   debounce: t.maybe(t.Integer)
 };
 
+export type ResizeSensorProps = {
+  /** content */
+  children: JSX.Element,
+  /** called when a resize event is captured */
+  onResize: () => void,
+  /** callback delay (milliseconds) */
+  debounce?: number
+};
+
+interface ResizeSensorElement extends Element {
+  resizedAttached?: {}
+};
+
 /**
  * A component used to intercept window resize events
- * @param children - content
- * @param onResize - called when a resize event is captured
- * @param debounce - callback delay (milliseconds)
  */
 @props(Props)
-export default class ResizeSensor extends React.Component {
+export default class ResizeSensor extends React.Component<ResizeSensorProps> {
+
+  private elementQueries: boolean
+  private resizeSensor: {} | null
 
   attachResizeSensor = () => {
     const { debounce } = this.props;
-    const element = ReactDOM.findDOMNode(this);
+    const element = ReactDOM.findDOMNode<ResizeSensorElement>(this);
```

Comment:
now `Element` is of type `ResizeSensorElement` so that TS doesn't complain on the next line.